src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/EditAppealAfterSubmitHandler.java

```diff
@@ -117,6 +118,18 @@ private void clearNewMatters(AsylumCase asylumCase) {
         }
     }
 
+    private void clearLitigationFriendDetails(AsylumCase asylumCase) {
+        YesOrNo hasLitigationFriend = asylumCase.read(LITIGATION_FRIEND, YesOrNo.class).orElse(NO);
```

Comment:
Need to clear phone number if wantsEmail, and clear email if wantsSMS

Reply:
Have added this into handler and test now